src/popup/components/Protocol/index.tsx

```diff
@@ -81,6 +87,15 @@ function ProtocolState() {
     return <OptInForm onOptIn={() => optInWithMnemonic()} />;
   }
 
+  const onComplete = () => {
+    getMnemonicSave().setCheckNotNeeded();
```

Comment:
This is a weird line, I think because "check" is usually used as a verb. Consider "setChallengeNotNeeded" and "setChallengeNeeded"?